pod.go

```diff
@@ -362,8 +362,12 @@ func lockPod(podID string) (*os.File, error) {
 		return nil, err
 	}
 
-	err = syscall.Flock(int(lockFile.Fd()), syscall.LOCK_EX)
-	if err != nil {
+	lockType := syscall.LOCK_SH
```

Comment:
I'd probably code this as an if/else - seeing something unconditionally set and then immediately conditionally set always makes me scrutinise the code as it looks there is something 'clever' going on to my eyes :-)

Reply:
done